platform/default/glfw_view.cpp

```diff
@@ -82,6 +82,20 @@ GLFWView::GLFWView(bool fullscreen_, bool benchmark_)
     pixelRatio = static_cast<float>(fbWidth) / width;
 
     glfwMakeContextCurrent(nullptr);
+
+    printf("\n");
+    printf("================================================================================\n");
+    printf("\n");
+    printf("- Press `S` to cycle through bundled styles\n");
+    printf("- Press `X` to reset the transform\n");
+    printf("- Press `N` to reset north\n");
+    printf("- Press `Tab` to toggle debug information\n");
+    printf("- Press `C` to toggle symbol collision debug boxes\n");
+    printf("- Press `Esc` to quit\n");
+    printf("- Press `1` through `6` to add increasing numbers of point annotations for testing\n");
+    printf("\n");
+    printf("================================================================================\n");
+    printf("\n");
```

Comment:
Whoa! Missed those. They've never been in the Markdown docs anyway! Case in point for having them in the same file as the actual keybinding definitions.